datadog/tests/cassettes/TestAccDatadogIntegrationAzure.yaml

```diff
@@ -227,7 +227,7 @@ interactions:
     method: GET
   response:
     body: |
-      [{"tenant_name":"aaaaaaaa-bbbb-cccc-dddd-1692885321ee","client_id":"testc7f6-1234-5678-9101-3fcbf464test","host_filters":"","app_service_plan_filters":"bar:baz,stinky:pete","automute":true,"cspm_enabled":true,"custom_metrics_enabled":true,"errors":[],"warnings":[]}]
+      [{"tenant_name":"aaaaaaaa-bbbb-cccc-dddd-1698416332ee","client_id":"testc7f6-1234-5678-9101-3fcbf464test","host_filters":"","container_app_filters":"bazinga:bazingo,woody:pride","app_service_plan_filters":"bar:baz,stinky:pete","automute":true,"cspm_enabled":true,"custom_metrics_enabled":true,"errors":["Invalid credentials. Datadog could not authenticate with Azure."],"warnings":[]}]
```

Comment:
Is this 'Invalid' because of missing client secret?

Reply:
this is invalid because we're using a fake tenant name/client id/etc to set up the integration. These tests are mainly to test that the api properly creates the integration, so we don't care if the integration itself doesn't work.

Reply:
hmm, its possible this is because of the tenant name change? not sure why that happened, maybe someone also removed the auth in the azure portal